devtools/platform-descriptor-json/src/main/resources/templates/README.maven.ftl

```diff
@@ -0,0 +1,29 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+## Running the application in dev mode
+
+You can run your application in dev mode that will allow to have hot reload feature using:
+```
+./mvnw compile quarkus:dev
```

Comment:
```suggestion
./mvnw quarkus:dev
```
compile is now done automatically if needed.

Reply:
Is compile also done automatically for `application.properties` file modification ?

Reply:
Yes, the invoker automatically compiles source code